### DISCUSSION TITLE
Edimax EW-7833UAC plug'n'play compatibility

### DISCUSSION THREAD

**Discussion by xauberer** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#37] [General]
Hi,

I'd like to report perfect plug'n'play compatibility with Edimax EW-7833UAC AC1750 on Open Core Big Sur. Original Edimax drivers are incompatible with Big Sur, and their windoze 10 drivers are really badly written. Your driver is sometimes even 20%-50% (!!) faster than original windoze counterpart, on the same PC and network.

Congratulations, many thanks and kudos!

**Comment by chris1111** [comment]
Thanks I will add to the Known testing and working adapter
